devpi/plugins/devpi-json-info/devpi_json_info.py

```diff
@@ -0,0 +1,68 @@
+import os
+from typing import Any
+
+from devpi_common.url import URL
+from devpi_server.readonly import get_mutable_deepcopy
+from pluggy import HookimplMarker
+from pyramid.view import view_config
+
+devpiserver_hookimpl = HookimplMarker("devpiserver")
+
+
+@devpiserver_hookimpl
+def devpiserver_pyramid_configure(config, pyramid_config):
+    # by using include, the package name doesn't need to be set explicitly
+    # for registrations of static views etc
+    pyramid_config.include("devpi_json_info")
+
+
+def includeme(config):
+    config.add_route("json_info", "/{user}/{index}/{project}/json")
+    config.scan()
+
+
+@view_config(
+    route_name="json_info",
+    accept="application/json",
+    request_method="GET",
+    renderer="json",
+)
+def json_info_view(context, request) -> dict[str, Any]:
+    # Override the base_url with the BASE_URL environment variable
+    # This is useful when running behind a reverse proxy
```

Comment:
Wait so the sha256 changes if run behind a reverse proxy? I would have guessed it shouldn't. 

Isn't the point to use stock devpi server?